src/components/input/Select.tsx

```diff
@@ -621,22 +628,25 @@ function SelectMain<T>({
             },
             listboxClasses,
           )}
-          role="listbox"
-          ref={listboxRef}
-          id={listboxId}
-          aria-multiselectable={multiple}
-          aria-labelledby={buttonId ?? defaultButtonId}
-          aria-orientation="vertical"
-          data-testid="select-listbox"
-          data-listbox-open={listboxOpen}
+          ref={popoverRef}
           // nb. Use `undefined` rather than `false` because Preact doesn't
           // handle boolean values correctly for this attribute (it will set
           // `popover="false"` instead of removing the attribute).
```

Comment:
Incidentally I think this might be fixed now, but it needs testing.

Reply:
Yep. I considered changing it, but I think we can tackle it individually